Curriculum Vitae.md

```diff
@@ -127,8 +127,7 @@ I was taken on as a junior engineer on the AHRLAC project during its prototyping
 	- An experiment in IoT written in Javascript, whereby a simple HTML5 Mandala-drawing canvas was provided, input from the user was published to an AWS topic, and then read by a Raspberry Pi, which controlled n-Spheros. The end goal is to do long-exposure/time-lapse aerial photography of the n-spheros developing a mandala that mimics the one the user drew.
 
 ##### Additional:
-
-I'm big into hiking and surfing, these are things that are pretty much a staple if you live in Cape Town, and play a lot of golf.  It helps me get used to managing frustration!
+I'm big into hiking, surfing and Roark, these are things that are pretty much a staple if you live in Cape Town, and play a lot of golf.  It helps me get used to managing frustration!
```

Comment:
Do you think use of the Oxford comma would be prudent here?

Reply:
It also seems like there's been a newline _removed_ now by accident - perhaps in effort to correct the one you added in the last commit.

Thanks for your efforts thus far @andrewlynch !

Reply:
The commas are a bit confusing now. I suggest using dashes 

`I'm big into hiking, surfing and Roark - these are things that are pretty much a staple if you live in Cape Town - and play a lot of golf.  It helps me get used to managing frustration!`

Reply:
Good thoughts on the dashes. 

Suggestion: Would golf not be considered a staple of Cape Town life?

`I'm big into hiking, surfing, Roark and golf - these are things that are pretty much a staple if you live in Cape Town. It helps me get used to managing frustration!`

Reply:
Hmmm @andrewlynch perhaps, but then the second sentence no longer makes sense. Perhaps we need to reword this section completely?

Reply:
@StuartHadfield perhaps we should take a step back and do some systems analysis on the activities you partake in. We all agree that this current iteration seems to be bloated, so I think it's prudent that we spec out what is required by your CVs end user and then set up a few sprints to work through the backlog.